### DISCUSSION TITLE
Why Aggregate and Window functions outside Functions section?

### DISCUSSION THREAD

**Discussion by jaceklaskowski** [duckdb/duckdb#7919] [General]
Just found it intriguing that [Aggregate Functions](https://duckdb.org/docs/sql/aggregates) and [Window Functions](https://duckdb.org/docs/sql/window_functions) are outside [Functions](https://duckdb.org/docs/sql/functions/overview) in the menu and thought I'd ask why.

Can anyone explain why the two function-related sections happen to be outside Function section?

<img width="231" alt="image" src="https://github.com/duckdb/duckdb/assets/62313/6ba5f709-0285-49ed-bfb7-77adeb576864">

**Comment by Tishj** [comment]
The docs have their own repo https://github.com/duckdb/duckdb-web

But to answer your question, I think everything under `Functions` deals with scalar functions, meaning functions that take a single tuple and produce a single tuple.

Aggregates are a different breed of function because they take multiple tuples (entire columns or subsets if filtered) as input and produce a single tuple as output.

Window functions also function much different from the regular scalar function, so they shouldn't be grouped together with them
